content/sensu-go/5.2/installation/auth.md

```diff
@@ -0,0 +1,442 @@
+Make sure to include the [group prefix](#groups-prefix) and [username prefix](#username-prefix) when creating Sensu role bindings and cluster role bindings.
+See the [RBAC reference][4] for a complete guide to creating permissions with Sensu.
+
+Once you've configured the correct roles and bindings, your users can log in to sensuctl and the Sensu dashboard using their single-sign-on username and password (no prefix required).
+
+### Configuration examples
+
+_NOTE: These examples are in `wrapped-json` format for use with [`sensuctl create`][sc]. To use the [authentication API][9], use `json` format as shown in the [API docs][9]._
+
+**Example LDAP configuration: Minimum required attributes**
+
+{{< highlight json >}}
+{
+  "Type": "ldap",
+  "api_version": "enterprise/v2",
+  "spec": {
+    "servers": [
+      {
+        "host": "127.0.0.1",
+        "binding": {
+          "user_dn": "cn=binder,dc=acme,dc=org",
+          "password": "P@ssw0rd!"
+        },
+        "group_search": {
+          "base_dn": "dc=acme,dc=org"
+        },
+        "user_search": {
+          "base_dn": "dc=acme,dc=org"
+        }
+      }
+    ]
+  },
+  "metadata": {
+    "name": "default"
+  }
+}
+{{< /highlight >}}
+
+**Example LDAP configuration: All attributes**
+
+{{< highlight json >}}
+{
+  "type": "ldap",
+  "api_version": "enterprise/v2",
+  "spec": {
+    "servers": [
+      {
+        "host": "127.0.0.1",
+        "port": 636,
+        "insecure": false,
+        "security": "tls",
+        "binding": {
+          "user_dn": "cn=binder,dc=acme,dc=org",
+          "password": "P@ssw0rd!"
+        },
+        "group_search": {
+          "base_dn": "dc=acme,dc=org",
+          "attribute": "member",
+          "name_attribute": "cn",
+          "object_class": "groupOfNames"
+        },
+        "user_search": {
+          "base_dn": "dc=acme,dc=org",
+          "attribute": "uid",
+          "name_attribute": "cn",
+          "object_class": "person"
+        }
+      }
+    ],
+    "groups_prefix": "ldap",
+    "username_prefix": "ldap"
+  },
+  "metadata": {
+    "name": "default"
+  }
+}
+{{< /highlight >}}
+
+## LDAP specification
+
+### Top-level attributes
+
+type         | 
+-------------|------
+description  | Top-level attribute specifying the [`sensuctl create`][sc] resource type. LDAP definitions should always be of type `ldap`.
+required     | Required for LDAP definitions in `wrapped-json` or `yaml` format for use with [`sensuctl create`][sc].
+type         | String
+example      | {{< highlight shell >}}"type": "ldap"{{< /highlight >}}
+
+api_version  | 
+-------------|------
+description  | Top-level attribute specifying the Sensu API group and version. For LDAP definitions, this attribute should always be `enterprise/v2`.
```

Comment:
I believe its now `authproviders/v2` (or at least that's the behavior on master). @palourde can you confirm?